src/dashboard/Push/PushIndex.react.js

```diff
@@ -141,15 +141,11 @@ let getPushName = (pushData) => {
     );
   } else {
     let payload = pushData[PushConstants.PAYLOAD_FIELD];
```

Comment:
You could just change this line to:  `let payload = pushData[PushConstants.PAYLOAD_FIELD] || '';`